beacon-chain/types/block.go

```diff
@@ -187,12 +195,18 @@ func (b *Block) IsValid(
 
 		for index, attestation := range b.Attestations() {
 			if !b.isAttestationValid(index, db, aState, cState, parentSlot) {
-				log.Debugf("attestation invalid: %v", attestation)
+				log.Errorf("attestation invalid: %v", attestation)
 				return false
 			}
 		}
 	}
 
+	cStateProposerRandaoSeed := cState.Validators()[proposerIndex].RandaoCommitment
+	if !b.isRandaoValid(cStateProposerRandaoSeed) {
+		log.Errorf("Invalid proposer RANDAO commitment. Wanted: %s. Got: %s", b.RandaoReveal(), cStateProposerRandaoSeed)
```

Comment:
this is a nitpick, but the error message should be `Wanted pre-image of  %#x, Got: %#x, cStateProposerRandaoSeed, b.RandaoReveal()`